lib/connection.js

```diff
@@ -1,12 +1,126 @@
 'use strict';
 
 const common = require('metarhia-common');
-const EventEmitter = require('events');
+const { EventEmitter } = require('events');
 
-const Connection = function() {
+const parser = require('./parser.js');
+
+const Connection = function(transport) {
   EventEmitter.call(this);
+
+  this._transport = transport;
+  this._parcels = [];
+  this._chunks = [];
+
+  // Mechanism for collecting full data,
+  // cut by lower-level protocols
+  this._bytesToRead = 36;
+  this._data = Buffer.alloc(0);
```

Comment:
What is `_data`? Buffer for what?

Reply:
This is buffer for packets cut by TCP. Need to rename this field.